src/services/GitHubService.ts

```diff
@@ -55,6 +56,28 @@ class GitHubService {
 
 		return [];
 	}
+
+	async getIssues(user: string, repo: string): Promise<GitHubIssue[]> {
+		const url = `https://api.github.com/repos/${user}/${repo}/issues`;
+
+		const { data } = await axios.get(url);
+
+		if (data.length !== 0) {
+			const issues = data.filter((issueAndPr: any) => !issueAndPr.pull_request)
+				.slice(0, 3)
```

Comment:
I feel as if we shouldn't do the slicing in here but in the command. Just in case we want to use this method in the future but need more than 3.